Property_map/include/CGAL/Dynamic_property_map.h

```diff
@@ -49,22 +49,20 @@ struct Dynamic_property_map {
     }
   }
 
-
-  friend reference get(const Dynamic_property_map& m, const key_type& k)
+  friend value_type get(const Dynamic_property_map& m, const key_type& k)
```

Comment:
`get` return type is `reference`. If you have an issue then it might be that `reference` is wrong